BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00420-01
\ No newline at end of file
+1.0.25-prerelease-00424-01
```

Comment:
@mellinoe -- we've decided to merge yours as is so PR #1214 can follow yours and update, so no need to update your PR.